### PR TITLE
docs: Update Running Feast in Production guide

### PR DESCRIPTION
**Pull request by achals**
Signed-off-by: Achal Shah <achals@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: docs/how-to-guides/running-feast-in-production.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Running Feast in production
+from pyspark.sql import SparkSession# Running Feast in production
```
Comment (felixwang9817) [inline docs/how-to-guides/running-feast-in-production.md:1]: stray

File: docs/how-to-guides/running-feast-in-production.md [hunk 2]
```diff
@@ -43,7 +51,37 @@ Different options are presented in the [how-to guide](structuring-repos.md).
...
+    - name: AWS_SECRET_ACCESS_KEY
+      valueFrom:
+        secretKeyRef:
+          name: aws-credentials
+          key: aws-secret-access-key
+```
+
+
+
+### 2.1. Manual materialization
```
Comment (felixwang9817) [inline docs/how-to-guides/running-feast-in-production.md:84]: should be 2.2, next is 2.3, etc.

File: docs/how-to-guides/running-feast-in-production.md (hunk 1 above)
Comment (achals) [reply docs/how-to-guides/running-feast-in-production.md:1]: fixed

File: docs/how-to-guides/running-feast-in-production.md (hunk 2 above)
Comment (achals) [reply docs/how-to-guides/running-feast-in-production.md:84]: oops

**Review by felixwang9817 — APPROVED** [review]
/lgtm

**Pull request merged**